GiniPayApiLib/Classes/Documents/Payments/ResolvedPaymentRequest.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ResolvedPaymentRequest.swift
+//  GiniPayApiLib
+//
+//  Created by Nadya Karaban on 05.05.21.
+//
+import Foundation
+/**
+ Struct for payment request response
+ */
+public struct ResolvedPaymentRequest: Codable {
+    public var requesterUri: String?
```

Comment:
Here requsterUri should always exist, so it shouldn't be nullable

Reply:
Thanks Alex